documentation/how-to/tracking.en.md

```diff
@@ -5,89 +5,91 @@ tx_slug: documentation_how-to_tracking
 
 # Tracking
 
-QField allows you to create points, lines, and polygons from tracking your position.
+:material-tablet: Fieldwork
 
-## Activate tracking
+QField allows you to create points, lines or polygons and track yourself in the background while working on another feature layer or while having the device in your pocket. To enable tracking, you need to ensure that your positioning is activated.
 
-:material-tablet: Fieldwork
+## Set-Up Tracking Session
 
-To activate tracking, you must first make sure the positioning is active. Then,
-open the side dashboard, long-press on a legend layer within which you want to
-save your tracks to, and select the *Setup tracking* button to configure
-the tracking session.
+1. Open the legend and press long on the layer which you want to use for tracking
+2. Select **Setup tracking** and a new window will appear showing the configuration settings
```

Comment:
@beanzmo , we should come up with a choice on whether we want to use paragraphs or bullet points, I feel ATM we're just hoping from one style to another and that's providing for a sense of inconsistency.

```suggestion
1. Open the dashboard and long press on the legend layer which you want to use for tracking
2. Select **Setup tracking** and a new popup will appear showing the configuration settings
```